### PR TITLE
Initialize repo and add first files.

### PR DESCRIPTION
**Pull request by jorainer**

### CODE REVIEW

**Comment by jorainer** [comment]
No code yet @philouail , mostly an update of the README to describe the repo contents and some general setup files.

File: LICENSE
Comment (philouail) [inline LICENSE:1]: why no license ?

**Review by philouail — APPROVED** [review]
all good to me, so I approved. 
I do have some curiosity question maybe you can explain later. 
what the `.Rbuildignore` does exactly and `_pkgdown.yml` ? I looked a bit into the files i need to have in the metabonaut repo and i'm a bit confused by everything ^^

File: LICENSE
Comment (jorainer) [reply LICENSE:1]: license is defined in the DESCRIPTION - we don't require a dedicated LICENSE file.

**Comment by jorainer** [comment]
.Rbuildignore defines files or folders that should be ignored by `R CMD build`, i.e. when the tar.gz of the package is being built. The github repo has more files than we want to be included in the R package.

The _pkgdown.yml defines parameters and settings for the packagedown package that builds the github pages.

**Pull request merged**